CreatubblesAPIClientIntegrationTests/Spec/Gallery/GalleriesResponseHandlerSpec.swift

```diff
@@ -38,7 +38,7 @@ class GalleriesResponseHandlerSpec: QuickSpec
             {
                 let request = GalleriesRequest(page: 1, perPage: 10, sort: .popular, userId: nil, query: nil)
                 let sender =  TestComponentsFactory.requestSender
-                waitUntil(timeout: 10)
+                waitUntil(timeout: 25)
```

Comment:
That is a good question. Some tests run fine with 10 sec timeout, however those that I changed where failing because it was too short. Maybe we could make something like timeout tiers, i.e.
: short = 10, medium = 25, long = 100. I'm not sure we really need that though.